[DO NOT MERGE] Added list support to VA policies

t-robalt opened the pull request:
Added support to get list of VA policies for servers and managed instance, including databases.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

t-robalt committed:
Added list support to VA policies

t-robalt committed:
Changed suffixes

AutorestCI commented:
# Automation for azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-ruby

AutorestCI commented:
# Automation for azure-sdk-for-java

AutorestCI commented:
# Automation for azure-sdk-for-go

t-robalt committed:
added missing examples

azuresdkci commented:
Can one of the admins verify this patch?

AutorestCI commented:
# Automation for azure-sdk-for-node

t-robalt commented:
@yaakoviyun @yoavfr

jianghaolu commented on specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/examples/DatabaseVulnerabilityAssessmentListByDatabase.json:
```diff
@@ -0,0 +1,30 @@
+        "serverName": "vulnerabilityaseessmenttest-6440",
+        "vulnerabilityAssessmentName": "default",
+        "api-version": "2018-06-01-preview"
+    },
```
Missing required parameter "databaseName".

t-robalt replied:
Fixed.

t-robalt committed:
Updated database example

t-robalt committed:
Changed operation name.

the suffix of "_ListBy..." should be a substring of the operation group, thus changed to Instance.
